use std::error::Error;

fn main() -> Result<(), Box<dyn Error>> {
    russet::run()
}

